projects/collection-editor-library/src/lib/components/editor/editor.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container [ngSwitch]="pageId">
   <ng-container *ngSwitchCase="'collection_editor'">
     <div class="add-to-library mb-10">
-      <lib-header *ngIf="collectionTreeNodes && collectionTreeNodes.data" [pageId]="pageId" [labelConfigData]="toolbarConfig" [collectionTreeNodes]="collectionTreeNodes" [buttonLoaders]="buttonLoaders" (toolbarEmitter)="toolbarEventListener($event)"> </lib-header>
+      <lib-header [pageId]="pageId" [labelConfigData]="toolbarConfig" [showComment]="showComment" [buttonLoaders]="buttonLoaders" (toolbarEmitter)="toolbarEventListener($event)"> </lib-header>
```

Comment:
Do we need a separate `Input`? can we include the `buttonLoaders` object? just a suggestion. otherwise, we will end up adding input of each button no?

Reply:
removed the `showComment`  input, included it inside `buttonLoders` input